python/docs/conf.py

```diff
@@ -197,9 +201,17 @@
 ]
 
 
+def on_missing_reference(app, env, node, contnode):
+    return contnode
```

Comment:
Control flow ceases here. Should there be code after this `return`?

Reply:
I'll just comment this out for now. At some point we'll have to patch this to avoid warnings in the docs build because there will be some missing references (e.g. links to thrust) but early in development I was just using this as a quick way to suppress warnings. Once we decide exactly what docs should look like it'll be easier to evaluate what references are really missing and what fixes are needed.

Reply:
I've updated this function now.